test/unit/org/apache/cassandra/index/sai/disk/v1/kdtree/LeafOrderMapTest.java

```diff
@@ -45,7 +47,7 @@ public void test() throws Exception
         var input = out.toIndexInput();
 
         final byte bits = (byte) DirectWriter.unsignedBitsRequired(array.length - 1);
-        LongValues reader = DirectReader.getInstance(new SeekingRandomAccessInput(input), bits);
+        LongValues reader = LuceneCompat.getDirectReaderInstance(new SeekingRandomAccessInput(input, ByteOrder.LITTLE_ENDIAN), bits, 0);
```

Comment:
Do we need to use `LuceneCompat` here?

Reply:
We don't technically need to, but I don't think it hurts anything to include it in the test since that is what we use in the rest of the code base. Do you have a specific reason we should prefer the `DirectReader`?